.github/run_erblint.sh

```diff
@@ -2,7 +2,15 @@
 
 shopt -s globstar
 
-bundle exec erblint decidim**/app/{cells,views}/**/*.erb
+CURRENT_BRANCH=$(git branch --show-current)
+DEFAULT_BRANCH=$(git symbolic-ref refs/remotes/origin/HEAD --short)
```

Comment:
I think this will contain the `origin/` prefix, right?

If so, the comparison below wouldn't work correctly (`"${CURRENT_BRANCH}" == "${DEFAULT_BRANCH}"`).

For me it returns `origin/master` (because of the fork) but I believe normally it would return `origin/develop` when you'd have to strip off the `origin/` prefix for it to work.

I think there was some miscommunication regarding using the `--short` flag before, I was actually just looking at the `symbolic-ref` documentation. But you should be able to strip it e.g. using [`basename`](https://linux.die.net/man/1/basename).

Reply:
Included